source/use-cases/sensitive-data-encryption.txt

```diff
@@ -530,6 +565,11 @@ full `JSON Schema for the Medco Medical Management System
 
       View the `helper code in Javascript
       <https://raw.githubusercontent.com/mongodb/docs-assets/DOCSP-json-schema-helper-and-json/JSONSchemaCreator.js>`_.
+   .. tab::
+      :tabid: python
+
+      View the `helper code in Python
+      <https://raw.githubusercontent.com/mongodb/docs-assets/DOCSP-json-schema-helper-and-json/json_schema_creator.py>`_.
```

Comment:
Could we use camel case for the `keyId` function parameter in this snippet to be consistent?

Reply:
Fixed:
* Switched `keyId` -> `key_id` for json_schema_creator.py

Reply:
Could you run the linter on this file?

Reply:
Done :)